datetime_formatter/__openerp__.py

```diff
@@ -0,0 +1,12 @@
+# -*- encoding: utf-8 -*-
+
+{
+    "name": "Date & Time Formatter",
+    "summary": "Helper functions to give correct format to date[time] fields",
+    "version": "1.0.0",
+    "author": "Odoo Community Association (OCA)",
+    "category": "Tools",
+    "depends": [
+        "base",
```

Comment:
Just see the same comment in other PRs, this is the reason. No problem to keep it :wink: